tests/Integration/Console/StatusCommandTest.php

```diff
@@ -29,7 +30,7 @@ protected function setUp(): void
 
         $this->migrator = $this->createMock(Migrator::class);
```

Comment:
Hey, can we revert the changes in the tests and just help Laravel to resolve the dependencies as follows:

```php
$this->migrator = $this->createMock(Migrator::class);
$this->app->instance(Migrator::class, $this->migrator);
```

i.e. all we need to do is to register mocked objects in the service container, this will fix the phpstan issue.